client/login/wp-login/index.jsx

```diff
@@ -324,20 +426,43 @@ export class Login extends Component {
 
 		const footer = (
 			<>
-				{ shouldRenderFooter && (
-					<LoginLinks
+				{ isSocialFirst && isWhiteLogin ? (
+					<LoginFooter lostPasswordLink={ this.getLostPasswordLink() } />
+				) : (
+					shouldRenderFooter && (
+						<LoginLinks
+							locale={ locale }
+							privateSite={ privateSite }
+							twoFactorAuthType={ twoFactorAuthType }
+							isWhiteLogin={ isWhiteLogin }
+							isP2Login={ isP2Login }
+							isGravPoweredClient={ isGravPoweredClient }
+							signupUrl={ signupUrl }
+							usernameOrEmail={ this.state.usernameOrEmail }
+							oauth2Client={ this.props.oauth2Client }
+							getLostPasswordLink={ this.getLostPasswordLink.bind( this ) }
```

Comment:
Do we really need to `bind` ?

Reply:
Sadly, yes, otherwise we would have a problem accessing `this.props` in the LoginLinks arrow component!